elasticsearch/api.go

```diff
@@ -8,24 +8,22 @@ import (
 	"net/http"
 	"net/url"
 
-	"github.com/pkg/errors"
-
+	rchttp "github.com/ONSdigital/dp-rchttp"
```

Comment:
I think all the functionality from this file (elasticsearch/api.go) should be moved to dp-elasticsearch library. But this might be out of the scope of this ask (in the past migrating code took a long time and we went over the task estimates). We might want to create a technical debt task to do this.

Reply:
Note: I wouldn't bock this PR for this reason. It was just a comment.